pkg/sdk/go/message_test.go

```diff
@@ -66,7 +66,7 @@ func TestSendMessage(t *testing.T) {
 			chanID: chanID,
 			msg:    msg,
 			auth:   "",
-			err:    errors.NewSDKErrorWithStatus(apiutil.ErrBearerKey, http.StatusUnauthorized),
+			err:    errors.NewSDKErrorWithStatus(errors.Wrap(apiutil.ErrBearerKey, apiutil.ErrValidation), http.StatusUnauthorized),
```

Comment:
which is the error wrapper? `apiutil.ErrValidation` or `apiutil.ErrBearerKEy` . In other sections of the code I see you use `apiutil.ErrValidation` . Maintain consistency

Reply:
This had to be switched in different contexts because of how `errors.Wrap()` works.
When error wrapping gets stacked, and there are multiple layers; the errors.Contains(err, ErrValidation) can have different results.
Here, I had to write it like this to pass the testcases. I've tried my best to keep these consistent but unfortunately, it gets messy.

Reply:
ok